src/effects/VST/VSTEffect.cpp

```diff
@@ -3607,6 +3608,18 @@ EffectSettings VSTEffect::MakeSettings() const
 
 VSTEffectValidator::~VSTEffectValidator() = default;
 
+
+VSTEffectValidator::VSTEffectValidator
+(
+   VSTEffectInstance&       instance,
+   EffectUIClientInterface& effect,
+   EffectSettingsAccess&    access,
+   wxWindow*                pParent
```

Comment:
The commit that defines the Instance adds a reference to Instance in the Validator.  It also adds three members that are not needed.  They only duplicate protected members of the base class.